src/lib/query/article.ts

```diff
@@ -12,6 +12,14 @@ export const getAllArticles = async () => {
   });
 };
 
+export const getLatestArticles = async () => {
```

Comment:
There is no reason to use arrow function here. It looks prettier and less typing

```
export async function getLatestArticle() {
  ...
}
```

It is better to name parameter to specify how many latest article you want to get. 

```
getLatestArticle({ limit: 6, offset: 0 }); 
```

In case we will do future pagination